autonomy/chain/tx.py

```diff
@@ -47,6 +47,8 @@
     "INTERNAL_ERROR: nonce too low",
     "Got empty transaction",
     "AlreadyKnown",
+    "ALREADY_EXISTS",
+    "already known",
```

Comment:
Why not check for all spelling variants?

Reply:
Working on a regex